src/components/Strategies/StrategyList.tsx

```diff
@@ -28,7 +30,13 @@ function StrategyList() {
       if (strategy.comingSoon) {
         comingSoonText!.classList.add('visible');
       } else {
-        history.push('/strategies/' + strategy.url);
+        if (user.apiKey) {
+          history.push('/strategies/' + strategy.url);
+        } else {
+          const store = window.localStorage;
```

Comment:
Again, do not access localStorage every time. Also, you suddenly named this `store` while in the other file it's `localStorage`. You need to pick ONE name and use it CONSISTENTLY. Also, `store` generally refers to redux type storage so do not use that anyway.